src/scripts/components/main/navigation/navigation.tsx

```diff
@@ -11,10 +11,20 @@ import Share from '../share/share';
 import {MenuIcon} from '../icons/menu-icon';
 
 import styles from './navigation.styl';
+import InfoBubble from '../info-bubble/info-bubble';
+import {languageSelector} from '../../../selectors/language';
 
 const Navigation: FunctionComponent = () => {
   const dispatch = useDispatch();
   const [showMenu, setShowMenu] = useState(false);
+  const [showInfoBubble, setshowInfoBubble] = useState(true);
```

Comment:
We don't want to show the bubble if there is already a language in local storage right?
Something like that?
```js
const savedLanguage = localStorage.getItem('language');
const [showInfoBubble, setshowInfoBubble] = useState(!Boolean(savedLanguage));
```